### PR TITLE
feat: container expected state

### PR DESCRIPTION
**Pull request by robot9706**
Add option to specify a successful deployment condition for each image. These options are the container states shown on the deployment instance list, with an extra option to define an exit code for the `Exited` state.

### CODE REVIEW

**Review by m8vago — CHANGES_REQUESTED** [review]
Let's move this to the docker config section.
Also please add a configuration option for timeout.

**Comment by polaroi8d** [comment]
Please fill out the pull request description.

File: golang/pkg/crane/k8s/deploy_facade.go [hunk 1]
```diff
@@ -17,6 +17,8 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 )
 
+const PodStateTimeoutSeconds = 120
```
Comment (nandor-magyar) [inline golang/pkg/crane/k8s/deploy_facade.go:20]: It seems unused, is it?

**Review by nandor-magyar — APPROVED** [review]
It looks good, apart from that single unused variable.

**Review by m8vago — APPROVED** [review]

**Pull request merged**